src/Pact/Types/Term.hs

```diff
@@ -931,10 +304,28 @@ instance Pretty n => Pretty (Lam n) where
 
 instance NFData n => NFData (Lam n)
 
-instance (ToJSON (Term n), FromJSON (Term n)) => ToJSON (Lam n) where toJSON = lensyToJSON 2
-instance (ToJSON (Term n), FromJSON (Term n)) => FromJSON (Lam n) where parseJSON = lensyParseJSON 2
```

Comment:
Stripping just 2 characters from the constructor fields names results in prefixing all properties with `am`. Is that intended?

Reply:
:+1: 

This was a small mistake during the release of lambdas